embedchain/llm/base.py

```diff
@@ -287,7 +285,7 @@ def chat(self, input_query: str, contexts: List[str], config: BaseLlmConfig = No
                 self.config: BaseLlmConfig = BaseLlmConfig.deserialize(prev_config)
 
     @staticmethod
-    def _get_messages(prompt: str, system_prompt: Optional[str] = None) -> List[BaseMessage]:
+    def _get_langchain_messages(prompt: str, system_prompt: Optional[str] = None) -> List[BaseMessage]:
```

Comment:
The updated name is more precise. As this function is expected to return langchains `BaseMessage` type and not our `ECChatMessage`. It will be confusing to keep the `_get_messages` and return langchains message type.